src/app/content/components/Toolbar.tsx

```diff
@@ -76,10 +53,24 @@ const SearchInputWrapper = styled.div`
   display: flex;
   align-items: center;
   margin-right: 4rem;
-  border-bottom: solid 0.1rem ${iconColor};
+  position: relative;
+
+  :after {
```

Comment:
What is this doing? It seems like it is just adding a line under the input box?

Reply:
yea it makes the underline, initially it used border-bottom but then the content was pushed up by 1px and i didn't want to make the container 1px bigger than everything else, with the right alignment, and overflow visible just to make the line show up